examples/core-auth-nextjs-pages/src/pages/auth/signin.tsx

```diff
@@ -49,6 +44,7 @@ export default function SignIn({
           return {};
         } catch (error) {
           // An error boundary must exist to handle unknown errors
+          console.log('Is this happening');
```

Comment:
I think you missed this console log, just saw it :D

Reply:
I can remove it in the release PR.

Reply:
Missed that, thanks for that @apedroferreira :)